lib/web/apiserver.go

```diff
@@ -480,6 +487,9 @@ func (h *Handler) bindDefaultEndpoints(challengeLimiter *limiter.RateLimiter) {
 	// Migrated this endpoint to /webapi/sessions/web below.
 	h.POST("/webapi/sessions", httplib.WithCSRFProtection(h.createWebSession))
 
+	// Forwards traces to the configured upstream collector
+	h.POST("/webapi/traces", httplib.MakeHandler(h.traces))
```

Comment:
I'm concerned about this being an unauthorized endpoint. What could we do to prevent it from being abused?

Reply:
Should we hold until that is done?

In other places I would typically bind observability endpoints to a separate port, so it's easier to not expose those to the internet when behind a Proxy/LB. In that spirit, we could have this endpoint unbound by default, add a (yet another?) config toggle and instruct admins to block wider access to it. It's not great, but it's a band-aid until we have something better.